### PR TITLE
LoTW SAT name should not be required

### PR DESCRIPTION
**Pull request by phl0**
The LoTW name should not be required. Based on https://github.com/wavelog/wavelog/pull/1303 which should be merged first.

### CODE REVIEW

File: assets/js/sections/satellite.js
Comment (int2001) [inline assets/js/sections/satellite.js:112]: This `&times;` produces a "x" in front of the alert-text. There's no spacing. Maybe add one?!

And: It has no effect in chrome (perhaps i am using it wrong - from reading the code i assume it should close the message?!)

Same at L59

**Review by int2001 — CHANGES_REQUESTED** [review]
Works and is logical. See comment (Cosmetics)

File: assets/js/sections/satellite.js
Comment (phl0) [reply assets/js/sections/satellite.js]: Added a space. The "X" closes the popup (on my Firefox system).

**Comment by int2001** [comment]
Looks better now. Closing the popup isn't working here (seems to be local problem)

**Comment by phl0** [comment]
> Looks better now. Closing the popup isn't working here (seems to be local problem)

Tnx. No issues with closing on Firefox here. Is a separate issue with Chrome which existed before obviously. Will merge after approval of https://github.com/wavelog/wavelog/pull/1303.

**Pull request merged**
